appservice/src/createAppService/AppInsightsCreateStep.ts

```diff
@@ -53,11 +55,43 @@ export class AppInsightsCreateStep extends AzureWizardExecuteStep<IAppServiceWiz
         return !wizardContext.appInsightsComponent;
     }
 
-    private async appInsightsSupportedInLocation(wizardContext: IAppServiceWizardContext, location: Location): Promise<boolean> {
+    public async appInsightsSupportedInLocation(wizardContext: IAppServiceWizardContext, location: Location): Promise<boolean> {
```

Comment:
How about you change this to return `string | undefined` instead of having a private `_location` variable. Return the location name if it's supported, otherwise undefined